### PR TITLE
Merge stuff from the team.

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

File: NewAuthMeConf/converters.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+_________________________________________________________________________________
+>                                                                                <
+>        WARNING: Conversions can't be undone! DO ALWAYS A BACKUP BEFORE!!!      <
+>                                                                                <
+_________________________________________________________________________________
+>                AuthMe Reloaded  converters  description  file                  <
+_________________________________________________________________________________
+
+Built-in converters (Into the Plugin's core module):
+        Old File Backend ("file") >>> SQLite    ("/authme converter flatfiletosqlite") THIS CONVERTER WILL BE REMOVED FROM THE CORE IN FUTURE VERSIONS!!!
```
Comment (Xephi) [inline NewAuthMeConf/converters.txt]: WUT ? it's /converter command my friend

Comment (sgdc3) [inline NewAuthMeConf/converters.txt]: ?
What do you mean?
He is your friend? xD I KILLED HIM :smiling_imp:

Comment (Xephi) [inline NewAuthMeConf/converters.txt]: To convert database, it's not /authme converter, but /converter

Comment (sgdc3) [inline NewAuthMeConf/converters.txt]: Yes this is only an idea for the future module

**Comment by sgdc3** [comment]
Now it's more clear ;)

File: PROTOTYPE AREA (future stuff and etc...)/converters.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+_________________________________________________________________________________
+>                                                                                <
+>        WARNING: Conversions can't be undone! DO ALWAYS A BACKUP BEFORE!!!      <
+>                                                                                <
+_________________________________________________________________________________
+>                AuthMe Reloaded  converters  description  file                  <
+_________________________________________________________________________________
+
+Built-in converters (Into the Plugin's core module):
+        Old File Backend ("file") >>> SQLite    ("/authme converter flatfiletosqlite") THIS CONVERTER WILL BE REMOVED FROM THE CORE IN FUTURE VERSIONS!!!
```
Comment (Xephi) [inline PROTOTYPE AREA (future stuff and etc...)/converters.txt]: Commands are still wrong

**Comment by sgdc3** [comment]
@xephi you don't understand. These are only prototypes and ideas for future versions. look at the directory name ;)

**Pull request merged**